rpm/0032-Ignore-Re-Fwd-when-sort-by-subject.patch

```diff
@@ -0,0 +1,77 @@
+diff --git a/src/libraries/qmfclient/qmailmessagesortkey.cpp b/src/libraries/qmfclient/qmailmessagesortkey.cpp
```

Comment:
It would be helpful if you could include a header similar to the other patches here, so that they can be applied using `git am`. The easiest way to generate the appropriate patch file that I'm aware of is to commit your changes to a temporary branch and then use `git format-patch -N HEAD~` to create the patch file.

Reply:
Fixed.